spring-boot-starter/example-autodeployment/README.md

```diff
@@ -36,7 +36,7 @@ public class AutoDeploymentApplication {
 
 3. You can also put BPMN, CMMN and DMN files in your classpath, they will be automatically deployed and registered within the process application.
 
-4. When implementing the process application in [standard manner](https://docs.camunda.org/manual/7.13/user-guide/process-applications/the-process-application-class/),
+4. When implementing the process application in [standard manner](https://docs.camunda.org/manual/7.14/user-guide/process-applications/the-process-application-class/),
```

Comment:
Should we simply go for `latest` here so it doesn't have to be upgraded every time?